### PR TITLE
Homepage Verbiage tweaks

### PR DESCRIPTION
**Pull request by brandonhundt**
A few content tweaks for the homepage. 

(Changes are NOT coupled with the CPB announcement; can go anytime.)

### CODE REVIEW

File: pages/home.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ image: /assets/img/og-image.jpg
     <div class="row">
       <div class="col-md-8">
         <div class="hero-content-inner">
-          <h1 class="mb-4 fw-bold">A privacy-first suite of applications for all your podcasting needs</h1>
+          <h1 class="mb-4 fw-bold">The professional publishing and monetization platform for your podcasting journey</h1>
```
Comment (kookster) [inline pages/home.md:17]: I don't love the word "journey" - and I kinda want podcast earlier in the phrase:
`The podcast publishing and monetization platform for professionals`?

File: pages/home.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ image: /assets/img/og-image.jpg
 
 <section class="section bg-x-blue text-white p-5">
   <div class="container col-xxl-8">
-    <p class="fs-4 mb-4">Dovetail from PRX supports over 270 podcasts and enables over 80 different organizations — to expand audience, monetize content, and ensure long-term sustainability of their podcasts, all while upholding a steadfast commitment to privacy.</p>
+    <p class="fs-4 mb-4">Dovetail from PRX is the all-in-one platform trusted by podcasting professionals. We help you expand your audience, monetize content, and ensure long-term sustainability—all while maintaining a steadfast commitment to privacy. From independent creators to industry leaders, Dovetail scales with you, supporting your podcast's journey every step of the way.</p>
```
Comment (kookster) [inline pages/home.md:29]: yeah, you have the journey idea here, we don't need it in both

File: pages/home.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ image: /assets/img/og-image.jpg
               {% picture thumbnail img/artwork/TED-talks-daily.jpg --alt TED Talks Daily %}
             </div>
             <div class="col d-flex align-items-center justify-content-center">
-              {% picture thumbnail img/artwork/themoth.png --alt The Moth %}
+              {% picture thumbnail img/artwork/normal-gossip.png --alt Normal Gossip %}
```
Comment (kookster) [inline pages/home.md:54]: 💯

File: pages/home.md (hunk 1 above)
Comment (brandonhundt) [reply pages/home.md]: Works for me. Updated!!!

**Pull request merged**